previews/primer/alpha/toggle_switch_preview.rb

```diff
@@ -62,6 +62,10 @@ def with_bad_csrf_token
       def with_turbo
         render(Primer::Alpha::ToggleSwitch.new(src: UrlHelpers.toggle_switch_index_path, turbo: true))
       end
+
```

Comment:
Should this be snapshotted? 🤔

Reply:
I would say yes, but only if it's visually different from other previews.

Reply:
The only visual difference would be whatever focus styles are in place. I've got aggressive settings on my local machine when it comes to focus, so it's always visually distinct, so I'm not sure if that's _actually_ true for this component as well or just me 😅  Lemme take a quick look at the CSS and/or disable my focus stuff to confirm.

Reply:
Ok, AFAICT ToggleSwitch has [focus styling](https://github.com/primer/view_components/blob/59933d4eb741d307657d87c042f2ebc4671ffc7e/app/components/primer/alpha/toggle_switch.pcss#L45-L48), but it's only visible when `:focus-visible` is applied via the browser. So I don't think we need a separate preview 👍